examples/website/src/shared/route/articleAdapter.ts

```diff
@@ -26,6 +27,24 @@ export function authorsAdapter(authors: any): Author[] {
   })
 }
 
+export function commentsAdapter(comments: any): Comment[] {
```

Comment:
Should this be typed more specifically?

Reply:
Good question, I don't know. I followed the examples from the other Adapters in the file (`authorsAdapter`, `articleMetaAdapter`, etc.)  and they also use `any` at this place. What do you think @roosnic1 ?

Reply:
It's always better to have more types but since this will likely change in the future anyway we can leave it at any,